reqwest-retry/Cargo.toml

```diff
@@ -18,11 +22,11 @@ futures = "0.3.0"
 http = "1.0"
 reqwest = { version = "0.12.0", default-features = false }
 retry-policies = "0.4"
-tracing = "0.1.26"
+tracing = { version = "0.1.26", optional = true }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 hyper = "1.0"
-tokio = { version = "1.6.0", features = ["time"] }
+tokio = { version = "1.6.0", default-features = false, features = ["time"] }
```

Comment:
There's a small risk that this could cause build failures, for people with `default-features = false` on their tokio dependency but also inadvertently depend on one or more of tokio's default features.

That being said, I think the risk is low enough to not require a breaking release. No need to change anything here